docs/src/testing/gas-etimation.md

```diff
@@ -60,13 +64,13 @@ Let's calculate gas based on those operations:
 - one storage write
 - one L2 -> L1 message (with `[1, 2, 3]` as a payload)
 
-We assume the cost of a single 32-byte word is 612 gas units. Said that we can estimate the cost of previous operations:
-- contract deployment - 3 * 612 = 1836
-- storage write - 2 * 612 = 1224
-- one L2 -> L1 message - (3 + 3) * 612 = 3672 (read more about L2 -> L1 message cost
+We assume the cost of a single 32-byte word is 551 gas units. Said that we can estimate the cost of previous operations:
+- contract deployment - 3 * 551 = 1653
+- storage write - 2 * 551 = 1102
+- one L2 -> L1 message - (3 + 3) * 551 = 3306 (read more about L2 -> L1 message cost
   [here](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism/#l2l1_messages))
 
-This time we sum all calculated values because all of them will be kept onchain. Overall onchain data gas cost is `6732`.
+This time we sum all calculated values because all of them will be kept onchain. Overall onchain data gas cost is `6061`.
```

Comment:
Not sure if this is correct, the cost will be much bigger, see https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism/#overall_fee

Reply:
I would write a test simulating it locally and check what the gas cost is and try to figure out with docs why it is this way

Reply:
The L2 L1 message cost is way bigger than before afaik